src/plugins/vis_types/timeseries/public/application/components/vis_types/timeseries/config.js

```diff
@@ -538,8 +538,8 @@ export const TimeseriesConfig = injectI18n(function (props) {
       <EuiFlexGroup gutterSize="s" responsive={false} wrap={true}>
         <EuiFlexItem grow={false}>
           <EuiFormRow
-            label={i18n.translate('visTypeTimeseries.timeSeries.overrideIndexPatternLabel', {
-              defaultMessage: 'Override Index Pattern?',
+            label={i18n.translate('visTypeTimeseries.timeSeries.overrideDataViewLabel', {
+              defaultMessage: 'Override data view?',
```

Comment:
@KOTungseth @gchaps here the data view should be capitalized as the Index Pattern was before?

Reply:
`index pattern` and `data view` are correct. We only capitalize at the beginning of a sentence or feature titles: https://elastic.github.io/eui/#/guidelines/writing#capitalization